src/probnum/randprocs/markov/integrator/_iwp.py

```diff
@@ -118,47 +118,49 @@ def __init__(
         forward_implementation="classic",
         backward_implementation="classic",
     ):
-        # initialise BOTH superclasses' inits.
-        # I don't like it either, but it does the job.
-        _integrator.IntegratorTransition.__init__(
-            self,
-            num_derivatives=num_derivatives,
-            wiener_process_dimension=wiener_process_dimension,
-        )
+
+        # aliases for easier readability below
+        n, d = num_derivatives, wiener_process_dimension
         continuous.LTISDE.__init__(
             self,
-            drift_matrix=self._drift_matrix,
-            force_vector=self._force_vector,
-            dispersion_matrix=self._dispersion_matrix,
+            drift_matrix=self._iwp_drift_matrix(n, d),
+            force_vector=self._iwp_force_vector(n, d),
+            dispersion_matrix=self._iwp_dispersion_matrix(n, d),
```

Comment:
these kinds of changes were necessary to remove the `wiener_process_dimension` argument from the integrator. I also like it better this way, because the actual matrix is now only stored in LTISDE._drift_matrix, and not as some (cached) property in IntegratedWienerTransition